articles/vmware/vmw-how-configure-ipsec-vpn.md

```diff
@@ -70,7 +70,7 @@ Before you start configuring IPsec VPN settings, you'll need to make a note of t
 
 ***
 
-## Configuring edge gateway IPsec VPN settings for an advanced gateway
+## Configuring edge gateway IPsec VPN settings
 
 To configure the IPsec VPN, you first need to configure the VPN settings on your edge gateway.
```

Comment:
The changes wrt 9.1 seem fine, but this line seems like an unnecessary tautology?